src/main/java/racing/domain/Car.java

```diff
@@ -0,0 +1,60 @@
+package racing.domain;
+
+import java.util.Objects;
+
+public class Car implements Comparable<Car> {
+
+    private static final int MAX_NAME_LENGTH = 5;
+
+    private final String name;
+    private int position;
+
+    public Car(String name) {
+        validate(name);
+        this.name = name;
+        this.position = 0;
+    }
+
+    private void validate(String name) {
+        validateSize(name);
+    }
+
+    private void validateSize(String name) {
+        if (name.length() > MAX_NAME_LENGTH || name.isEmpty()) {
+            throw new IllegalArgumentException("자동차 이름 길이가 초과되었습니다.");
+        }
+    }
+
+    public void move() {
+        this.position++;
+    }
+
+    public boolean isSame(Car car) {
+        return this.position == car.position;
+    }
```

Comment:
저만의 소소한 방식인데, 
```suggestion
    public boolean isSame(Car other) {
        return this.position == other.position;
    }
```
이렇게 쓰면 this & other 가 반대되는 느낌이라 뭔가 보기 좋더라구요 :)
그냥 TMI...